asb-ballerina/Package.md

```diff
@@ -1,473 +1,22 @@
-# Ballerina Azure Service Bus Module
+Connects to Microsoft Azure Service Bus from Ballerina.
 
-Connects to Microsoft Azure Service Bus using Ballerina.
+## Package overview
 
-# Module Overview
-Microsoft Azure Service Bus is a fully managed enterprise message broker with message queues and publish-subscribe 
-topics. The primary wire protocol for Service Bus is Advanced Messaging Queueing Protocol (AMQP) 1.0, an open ISO/IEC 
-standard. Service Bus provides a Microsoft supported native API and this module make use of this public API. 
-Service Bus service APIs access the Service Bus service directly, and perform various management operations at the 
-entity level, rather than at the namespace level (such as sending a message to a queue). This module supports these 
-basic operations. These APIs use SAS authentication and this module supports SAS authentication.  
+The `ballerinax/asb` is a [Ballerina](https://ballerina.io/) connector for Microsoft Azure Service Bus.
 
-# Compatibility
-|                     |    Version                  |
-|:-------------------:|:---------------------------:|
-| Ballerina Language  | Swan-Lake-Alpha5            |
-| Service Bus API     | v3.5.1                      |
+This package provides the capability to easily access Microsoft Azure Service Bus to send and receive messages. You can perform actions such as send to queue, send to topic, receive from queue, receive from subscription, etc.
```

Comment:
You can perform operations to send and receive messages from Service Bus queues, topics, and subscriptions

Reply:
Resolved and amended the changes. Thank you for the suggestion.